build/version.go

```diff
@@ -1,9 +1,41 @@
 package build
 
 var CurrentCommit string
+var BuildType int
+
+const (
+	BuildDefault      = 0
+	BuildMainnet      = 0x1
+	Build2k           = 0x2
+	BuildDebug        = 0x3
+	BuildCalibnet     = 0x4
+	BuildInteropnet   = 0x5
+	BuildButterflynet = 0x7
+)
+
+func BuildTypeString() string {
+	switch BuildType {
+	case BuildDefault:
+		return ""
```

Comment:
```suggestion
```
Not sure what `default` stands for here, maybe just remove and bundle under `huh?`?